convert2rhel/actions/post_conversion/rhsm_custom_facts_config.py

```diff
@@ -0,0 +1,39 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+from convert2rhel import actions, subscription
+
+
+class RHSMCustomFactsConfig(actions.Action):
+
+    id = "RHSM_CUSTOM_FACTS_CONFIG"
+
+    dependencies = ()
+
+    def run(self):
+        super(RHSMCustomFactsConfig, self).run()
+
+        ret_code, output = subscription.update_rhsm_custom_facts()
+
```

Comment:
```suggestion
        ret_code, output = subscription.update_rhsm_custom_facts()
        if not output:
            return

```

Reply:
I'd say `return None` to be explicit

Reply:
@Andrew-ang9 After adding this suggestion the PR should be good for approval